src/test/java/org/gridsuite/securityanalysis/server/SecurityAnalysisControllerTest.java

```diff
@@ -188,11 +192,39 @@ public void tearDown() {
             .expectStatus().isOk();
     }
 
+    @SneakyThrows
+    public void simpleRunRequest(SecurityAnalysisParametersInfos lfParams) {
+        webTestClient.post()
+                .uri("/" + VERSION + "/networks/" + NETWORK_UUID + "/run?contingencyListName=" + CONTINGENCY_LIST_NAME_VARIANT + "&variantId=" + VARIANT_3_ID)
+                .bodyValue(lfParams)
+                .exchange()
+                .expectStatus().isOk()
+                .expectHeader().contentType(MediaType.APPLICATION_JSON)
+                .expectBody(SecurityAnalysisResult.class)
+                .value(new MatcherJson<>(mapper, RESULT_VARIANT));
+    }
+
+    @Test
+    @SneakyThrows
+    public void runTestWithLFParams() {
+        // run with some empty params
+        simpleRunRequest(SecurityAnalysisParametersInfos.builder().build());
```

Comment:
You are testing the run with different parameters but not really check anything specific.

Reply:
yes, for test coverage.
I did not see what else I could test.